.travis.yml

```diff
@@ -1,10 +1,15 @@
 dist: xenial
 language: c
+env:
+  matrix:
+    - BUILD_NAME="default"
+    - BUILD_NAME="utmp" FORCE_UTMP="1"
 compiler:
  - gcc
  - clang
 script:
  - echo "${CC} -O2 -Wall" > conf-cc
+ - if [ -n "${FORCE_UTMP}" ]; then cp qtmp.h1 qtmp.h; touch -t 203801012359 qtmp.h; fi
```

Comment:
Can we use make -o qtmp.h instead of a touch with a future date?

Reply:
Yes, @DerDakon provided this and it runs correctly green in Travis.